definitions/npm/mersenne-twister_v1.x.x/flow_v0.33.x-/mersenne-twister_v1.x.x.js

```diff
@@ -0,0 +1,18 @@
+declare module 'mersenne-twister' {
+  declare class $npm$MersenneTwister {
```

Comment:
You can try doing that... if you have `declare var exports: ...` it will assume it's a commonjs module and will not automatically export all inner declares (I am pretty sure... did not test it tho)